front-end/src/screens/MotionPost/query.ts

```diff
@@ -67,6 +67,7 @@ const motionPost = gql`
         }
         content
         created_at
+        has_poll
```

Comment:
```suggestion
```

Reply:
typescript complaining if it is not there